WebdriverIO.md

```diff
@@ -154,17 +154,20 @@ If you use one of options 2-4, you should create a chain of element selectors wh
 Then you can select Element B with this selector chain:
 
 ```js
-var elemB = $('.webdriverio-test-elem-a').$('#elem-b');
+var elemB = $('.e2e-test-elem-a').$('#elem-b');
 ```
 
 Notice that the top of the chain, where we select Element A, uses method 1.
 
 Sometimes you need to distinguish between several different elements which all look the same to element selectors. You can iterate over all the elements to find the right one. For example, suppose we want to click on the button to open a topic, where the button text is the topic name. We could find the right button like this:
 
 ```js
-var buttons = $$('.webdriverio-test-button');
+var buttonsSelector = function() {
+  return $$('.e2e-test-button');
+}
```

Comment:
Wouldn't it be good if you explain this too a bit in terms of flakiness like why we need a function for all selector when it's declared at the top?

Reply:
Done.